package br.com.excecoes.implementacao;

import tratarexcecao.MinhaExcecao;

public class TesteExcecao {

	public static void main(String[] args) {

		try {
		int numero = Integer.parseInt("7");
		if (numero<0) {
			throw new RuntimeException ("Ligue para o 9999");
		}
		System.out.println("Número: " + numero);
		
		String palavra = null; // teste positivo: String palavra = "";
		System.out.println("Qtde caracteres: " + palavra.length());	
		
		int[] valores = new int [2];
		valores[0] = 541;
		valores[1] = 82;
		valores[2] = 10;
		
				} catch (Exception e) {
			System.out.println(MinhaExcecao.tratar(e));
		} finally {
			System.out.println("Tenha um bom dia de trabalho!"); 
		}
		
		
		
		
		
	}

}
